ui/packages/catalog-ui-search/src/main/webapp/component/announcement/CommonMessages.tsx

```diff
@@ -14,10 +14,9 @@
  **/
 const properties = require('../../js/properties.js')
 
-export const InvalidSearchFormMessage = {
+export const InvalidSearchFormMessage: any = {
```

Comment:
:pencil2: Would be nice to typed this out

Reply:
I like this idea. I'm not too familiar with creating types in typescript, or if we as a team have a convention - would I do this by declaring an interface? And then doing something like this - https://zohaib.me/creating-object-based-on-interface-type-in-typescript/

Reply:
Upon further investigation it looks like this might be a little complicated to do. If I were to create a new type for it, I would probably want to include the `type` property as defined in `InvalidSearchFormMessage` . However, this would mean I would have to either assign that type everywhere where I'm constructing the message object with `title` and `body` , or I would have to use the de-structuring as you suggested. In the latter case, I believe I would have to import the `InvalidSearchFormMessage` in all of those places, which I was able to take out in the current implementation. I do like the idea, but perhaps its best saved for a separate refactoring task.